src/pages/home/_trade-types.tsx

```diff
@@ -249,13 +272,15 @@ const TradeTypes = (): React.ReactNode => {
                 mb="40px"
                 tablet={{ mb: '16px' }}
             >
-                {localize('Trade the way you want with 3 exciting trade types')}
+                {is_row
+                    ? localize('Trade the way you want with 3 exciting trade types')
+                    : localize('Trade the way you want with 2 exciting trade types')}
             </Header>
 
             <DesktopWrapper>
                 <Flex>
                     <Carousel {...settings}>
-                        {ItemsDetails.map((item) => {
+                        {(is_row ? items_details_cr : items_details_eu_uk).map((item) => {
```

Comment:
There are repetition of this statement here and at line 295. Best to set it once and re-use it in both lines.